src/repositories/user-repository.js

```diff
@@ -25,13 +25,13 @@ module.exports = function getUserRepository(sequelize) {
 
   async function returnUser(form) {
     const student = await selectUserDB(form, Student);
-    if (student) return { role: roles.STUDENT, user: student };
+    if (student) return [roles.STUDENT, student.dataValues];
 
     const teacher = await selectUserDB(form, Teacher);
-    if (teacher) return { role: roles.TEACHER, user: teacher };
+    if (teacher) return [roles.TEACHER, teacher.dataValues];
 
     const admin = await selectUserDB(form, Admin);
-    if (admin) return { role: roles.ADMIN, user: admin };
+    if (admin) return [roles.ADMIN, admin.dataValues];
```

Comment:
Why would you need the explicit `dataValues` ? there is a special `toJSON()` method that gives back the values.